### DISCUSSION TITLE
Disable stat messages

### DISCUSSION THREAD

**Discussion by jialonso** [arendst/Tasmota#10834] [Q&A] [answered]
Hi!

I have a Wemos with a CO2 sensor and a display. I have a rule triggered with every meassure of the sensor (every second) that updates the display to show CO2 readings. 

My problem is that each DisplayText command (performed by the rule) send a MQTT message to the stat/<topic>/RESULT and I want to disable this behavior. Is that possible? It would be perfect to add a flag to the DisplayText command to disable result reporting.

Best regards.

**Comment by sfromis** [comment, marked as answer]
No way of disabling that. Just ignore messages you have no use for.

**Comment by jialonso** [comment]
Thank you for your answer.

The real problem is that I use AWS IoT as MQTT broker and pricing is based on number of messages. One message every second is more than 2 million messages each month and my idea is to install a few of these devices. 

Thus, I will just remove the screen while I found a better alternative.

**Reply by ascillato** [reply]
The best option will be always to use a local mqtt server. A Raspberry pi with mosquitto for example. That is super reliable. And if you want safe external access you can install to that Raspberry home assistant or OpenHab.

**Reply by jialonso** [reply]
I have a raspberry pi at home, running mosquitto, zigbee2mqtt and homeassistant. All my sensors at home connect to the local mosquitto server running on the pi.

In addition to that, I have my mosquitto server bridged with AWS. My "external" sensors (at my parents's home, etc) connect directly to AWS IoT and I receive them at home through the bridge. That way I have all the sensor data in homeassitant as well as being able to run some small applications in the cloud.
